modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CustomViewsControls.js

```diff
@@ -57,8 +64,52 @@ const CustomViewsControls = () => {
 				},
 				{
 					label: Liferay.Language.get('save'),
-					onClick: ({processClose}) => {
-						processClose();
+					onClick: () => {
+						const url = new URL(`${appURL}/fds/${id}/custom-views`);
+
+						const data = {
+							name: customViewNameInputRef.current.value,
```

Comment:
Nice! :)

taking a look quickly I was going to create a state but changing this input will not cause any effect on the UI so we can use Refs, right?

Reply:
Yep. There is no need for a component refresh, that the state change would cause.